content/documentation/distributed-tracing/index.adoc

```diff
@@ -20,51 +20,40 @@ toc::[]
 
 Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+icon:bullhorn[size=2x]{nbsp} To have the tracing integration, you must set the configuration in external services / tracing.
 
-== Kiali and Tracing Service have the same protocol
+== Tracing url set
 
-When Kiali and Tracing service have the same protocol, we'll have the embedded feature in the website.
+When Tracing url is set, Kiali show a link to the Distributed Tracing in the menu.
```

Comment:
Once the `url` setting is set on the [Kiali CR](https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml#L448), Kiali **will** show...